test/config.test.js

```diff
@@ -38,30 +38,18 @@ it('Query info', () => {
           suite: 'wasm-misc',
           option: 'opt',
           platform: 'linux1804-64-shippable-qr',
-          platformSuffix: '-qr',
+          platformSuffix: undefined,
```

Comment:
Can this field be missing instead of undefined?

Reply:
The test fails if I do that.